src/wi_stuff.c

```diff
@@ -900,9 +913,13 @@ static void WI_initAnimatedBack(boolean firstcall)
   }
 
   if (exitpic)
-    return;
-  if (enterpic && entering)
```

Comment:
`entering` is a pointer to a patch, right? Why was this check here in the first place?

Reply:
It's a function parameter in PrBoom+: https://github.com/coelckers/prboom-plus/blob/969515162c5aebea4ad7a125ee178dcad3d576ad/prboom2/src/wi_stuff.c#L655

I accidentally copied it from PrBoom+.